src/command.rs

```diff
@@ -531,14 +546,10 @@ impl DefaultConfigurationValues for RelayChainCli {
 		30334
 	}
 
-	fn rpc_ws_listen_port() -> u16 {
+	fn rpc_listen_port() -> u16 {
```

Comment:
**MAYBE CR**

Shouldn't we get those values from the config? Ahh not possible... okay...